mybooks/oauth_views.py

```
@@ -18,6 +18,11 @@
 from mybooks.utils import build_code_challenge, get_code_verifier, get_oauth_server_metadata
 
 
+def oauth_metadata(request):
```

Ruff (0.13.1)

Unused function argument: `request`

(ARG001)